### DISCUSSION TITLE
How to edit every voxel's color?

### DISCUSSION THREAD

**Discussion by xing-yuu** [isl-org/Open3D#5783] [Q&A]
I create a voxel grid from a mesh. The voxel grid does not have color information. Now, I want to edit every voxel's color. What can I do? Thanks!

**Comment by hllllkkkkkk** [comment]
Hi, If you are creating voxel grid from a point cloud, you may try to add colored sphere to positions of voxels in your point cloud that you want to create your voxel grid from. I guess coloring a mesh would be much easier, also you can save the file in .off format.
